docs/helm/examples/external-hazelcast.md

```diff
@@ -0,0 +1,190 @@
+            <endpoint-group name="DATA" enabled="true"/>
+        </rest-api>
+    </network>
+    <jet enabled="true">
+    </jet>
+    ```
+
+5. Accordingly change the values for management center. Put internal network
+   address of your k8s.
+
+    ```xml
+    <management-center data-access-enabled="true">
+        <trusted-interfaces>
+            <interface>10.42.0.*</interface>
+        </trusted-interfaces>
+    </management-center>
+    ```
+
+6. Now preparation of the xml configuration is finished. Next step is to create
+   configmap manifest. Add below section to newly created file:
+
+    ```yaml
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: manual-hazelcast-configuration
+      labels:
+        app.kubernetes.io/name: hazelcast
+    data:
+      hazelcast.xml: |
+    ```
+
+    Now copy generated xml configuration into manifest file. (Remember to
+    correctly indent copied text)
+
+7. Deploy created [manifest](external-hazelcast_files/configmap-hazelcast.yaml)
+
+    ```bash
+    kubectl apply -f configmap-hazelcast.yaml 
+    ```
+
+8. Deploy hazelcast with [values](external-hazelcast_files/hazelcast.yaml)
+   presented below
+
+    ```yaml
+    hazelcast:
+      javaOpts: -Dhazelcast.config=/data/hazelcast/hazelcast.xml
+      existingConfigMap: manual-hazelcast-configuration
+    mancenter:
+      enabled: true
+        yaml:
+          hazelcast-client:
+            cluster-name: test
+    ```
+
+    ```bash
+    helm repo add hazelcast https://hazelcast-charts.s3.amazonaws.com/
+    helm repo update
+    helm install hazelcast hazelcast/hazelcast -f hazelcast.yaml
+    ```
+
+9. Now change the config of alfresco-repository by adding another [values
+   file](external-hazelcast_files/acs-hazelcast.yaml). In below file specify
+   properties that will make repository use external hazelcast cluster deployed
+   in previous step. Remember to accordingly change the values if needed.
+
+    ```yaml
+    alfresco-repository:
+      replicaCount: 3
+      config:
+        repository:
+          additionalGlobalProperties: 
+            alfresco.hazelcast.embedded: false
+            alfresco.hazelcast.client.address: hazelcast:5701
+            alfresco.cluster.name: test
+    ```
+
+    ```bash
+    helm upgrade acs helm/alfresco-content-services \    
+    --values local-dev-values.yaml \
+    --set global.search.sharedSecret=$(openssl rand -hex 24) \
+    --atomic \
+    --timeout 10m0s \
+    --values acs-hazelcast.yaml
+    ```
+
+## Tests
+
+1. Make sure that you have installed fresh environment. Do not login into
+   Alfresco UI yet.
+
+2. Add ubuntu deployment.
+
+    ```bash
+    helm repo add open https://simonmisencik.github.io/helm-charts
+    helm repo update
+    helm install testing open/ubuntu
+    ```
+
+3. Get into a shell of container created by previous commands.
+
+    ```bash
+    kubectl exec --stdin --tty testing-ubuntu -- /bin/bash
+    ```
```

Comment:
better use standard and well known stuff

no need for ubuntu helm chart, just use:

```
kubectl run -i --tty --rm --image ubuntu test-shell -- /bin/bash
```